R/pkg/R/sparkR.R

```diff
@@ -93,7 +93,7 @@ sparkR.stop <- function() {
 #' sc <- sparkR.init("local[2]", "SparkR", "/home/spark",
 #'                  list(spark.executor.memory="1g"))
 #' sc <- sparkR.init("yarn-client", "SparkR", "/home/spark",
-#'                  list(spark.executor.memory="1g"),
+#'                  list(spark.executor.memory="4g", spark.driver.memory="2g"),
```

Comment:
Yeah we should take this out of here. I think we should add a new section in the Programming guide called 'Running SparkR from RStudio' and include this option there. Here we could just have a link to the latest programming guide with something like 'For more options on how to initialize and use SparkR see our programming guide'